spec/models/user_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:user_1) do
```

Comment:
Use one line record with `{ }` avoid `do end` block in the test for let

Reply:
fix